fabfile.py

```diff
@@ -150,6 +152,15 @@ def bootstrap():
 
         sudo('apt-get update --fix-missing')
 
+    with settings(user=env.user):
```

Comment:
I think fabric has some kind of helper function for that (I believe it's called line). Take a look ;)